apps/mark-scan/frontend/src/constants.ts

```diff
@@ -1,4 +1,4 @@
 /* istanbul ignore file */
-export const STATE_MACHINE_POLLING_INTERVAL_MS = 300;
+export const STATE_MACHINE_POLLING_INTERVAL_MS = 200;
 // Overrides shorter libs/ui polling interval
-export const AUTH_STATUS_POLLING_INTERVAL_MS_OVERRIDE = 300;
+export const AUTH_STATUS_POLLING_INTERVAL_MS_OVERRIDE = 400;
```

Comment:
Could this be addressed by changing the precedence order that screens are rendered in? I'm not sure this solution of changing the polling intervals will be reliable enough.

Reply:
Unfortunately the "Insert card" screen is already the last screen in the render order. The logic can be summarized as `if (any auth including logged out) && state === 'empty_ballot_box' then render EmptyBallotBoxPage, else if logged out then render InsertCardPage`. So if the frontend sees `authStatus` is `logged_out` the only thing that will avoid flicker is if the frontend already has the updated state.

If we think the polling interval change is too hacky, we could change the last few states of the machine. I'll push a commit with that fix and we can see if it feels more reliable without too much complexity added.

Reply:
[357453a](https://github.com/votingworks/vxsuite/pull/4592/commits/357453a8650bb0b143ce6f621c9c0b7147761f92) renders `BallotSuccessfullyCastPage` (the screen for the previous state) regardless of auth status to avoid the flicker.